Remove license classifier

While trying to upgrade the debian package, I'm prompted with the following error:

License classifiers have been superseded by license expressions (see https://peps.python.org/pep-0639/). Please remove:

License :: OSI Approved :: BSD License
make[1]: *** [debian/rules:21: override_dh_auto_install] Error 1


It seems license classifiers have been dropped in PEP-639, and have been superseeded by license and license-files properties, which are already present in the pyproject.toml

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -17,6 +17,7 @@ Minor changes
 .............
 
 - Fix author email format. [:issue:`99` by @gboutry and @stevepiercy]
+- Drop deprecated license classifier. [:issue:`101` by @gboutry]
 
 
 Version 2.0.0

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,7 +29,6 @@ classifiers = [
     "Framework :: Sphinx :: Domain",
     "Framework :: Sphinx :: Extension",
     "Intended Audience :: Developers",
-    "License :: OSI Approved :: BSD License",
     "Operating System :: OS Independent",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3.10",

